Readme.md

```diff
@@ -1,33 +1,47 @@
 # World Explorer
 
-GM tool for hexcrawl campaigns that allows displaying a second manual fog of war (with color or image) only over the background layer, while keeping the grid, tokens, and tiles visible. Grid tiles can be removed manually by the GM to reveal the underlying map on successing scouting or mapping checks. Enable in scene configuration.
+GM tool for hexcrawl campaigns that allows displaying a second manual fog of war (with color or image) only over the background layer while keeping the grid visible. You can choose what else this layer covers or not: drawings, (overhead) tiles, and tokens. Grid spaces can be revealed or made partially visible. The GM can always do this manually, but you can also enable auto-revealing areas explored by friendly tokens.
 
 If you're feeling generous, you can send something through [Paypal](https://paypal.me/carlosfernandez1779?locale.x=en_US) if you want.
 
-![image](https://github.com/user-attachments/assets/a70120ac-b992-493e-94dc-c2569d8351c5)
-
 ## Features
 
-The module can be set to be enabled for a scene via the last tab in the scene settings. Once enabled, the button for it will show up on the canvas controls to the left. From there, you can either the map in toggle mode (where you enable/disable tiles one at time) or in reveal or hide modes. You can also reset the entire map from there (either hiding everything or showing everything).
+### Scene Settings & Canvas Tools
 
-![image](https://github.com/user-attachments/assets/814debd6-aab8-4e69-b355-d3f2dfc9d42b)
+<img src="images/screenshot-world-explorer-scene-settings.webp" title="World Explorer tab in the scene setting" width="60%" align="left">
+<img src="images/screenshot-world-explorer-tools.webp" title="World Explorer canvas control tools" width="30%" align="right">
+<br clear="right"/>
+<sub><br>Scene setting tab (left) and <br>Canvas control tools (right)</sub>
+<br clear="both"/>
+<br>
 
-### Separate Player and Gamemaster Opacities
+Enable this module in scene settings. Once enabled, a tool button will be available in the canvas controls to the left. With those, you can either edit the map in toggle mode (where you hide/reveal grid spaces one at time) or in reveal, partial, or hide modes. The tools also allow you to quickly change the opacity for the GM and to reset the entire map (hiding or revealing everything).
```

Comment:
```suggestion
Enable this module for a scene in the scene's configuration. Once enabled, a tool button will be available in the canvas controls to the left. With those, you can either edit the map in toggle mode (where you hide/reveal grid spaces one at time) or in reveal, partial, or hide modes. You can also quickly change the opacity for the GM and reset the entire map (hiding or revealing everything).
```

I'm not 100% of the first sentence, but I think we need to state somehow that its the scene config of a scene. I fear some people might think that scene settings are a global thing or something.